Mergin/ui/ui_switch_datasources_select_updated_project.ui

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>WizardPage</class>
+ <widget class="QWizardPage" name="WizardPage">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>400</width>
+    <height>249</height>
+   </rect>
+  </property>
+  <property name="windowTitle">
+   <string>WizardPage</string>
+  </property>
+  <layout class="QVBoxLayout" name="verticalLayout">
+   <item>
+    <widget class="QLabel" name="label">
+     <property name="text">
+      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The resulting QGIS project should be stored outside of the Mergin project folder.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
This probably should not need to be written there, and we would simply show a warning message box if the user tries to store it in a MM project directory...